src/main/java/Duke.java

```diff
@@ -1,10 +1,208 @@
+                    }
+                }
+                /** To add Deadline to list */
+
+                else if ((inputCommand.contains("deadline"))) {
+                    try {
+                        String[] attributeFinder = splitInputs[1].split("/by");
+
+                        String deadlineName = attributeFinder[0];
+                        String deadlineBy = attributeFinder[1];
+                        newTask(new Deadline(deadlineName, deadlineBy));
+                        System.out.println("____________________________________________________________");
+                        System.out.println("\tTask added:\t" + listTasks.get(jobCount - 1).printDetails());
+                        System.out.println("\tNow you have " + jobCount + " tasks in the list.\n");
+                        System.out.println("____________________________________________________________");
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println("____________________________________________________________\n" +
+                                "☹ OOPS!!! The description of a " + splitInputs[0] + " cannot be empty.\n" +
+                                "____________________________________________________________\n");
+                    }
+                }
+                /** To add event to list */
+                else if ((inputCommand.contains("event"))) {
+                    try {
+                        String[] attributeFinder = splitInputs[1].split("/at");
+                        String eventName = attributeFinder[0];
+                        String eventAt = attributeFinder[1];
+                        newTask(new Events(eventName, eventAt));
+                        System.out.println("____________________________________________________________");
+                        System.out.println("\tTask added:\t" + listTasks.get(jobCount - 1).printDetails());
+                        System.out.println("\tNow you have " + jobCount + " tasks in the list.\n");
+                        System.out.println("____________________________________________________________");
+                    } catch (IndexOutOfBoundsException e) {
+                        System.out.println("____________________________________________________________\n" +
+                                "☹ OOPS!!! The description of a " + splitInputs[0] + " cannot be empty.\n" +
+                                "____________________________________________________________\n");
+                    }
+                } else {
+                    try {
+                        gibberishError();
+                    } catch (NoSuchCommandException e) {
+                        System.out.println("____________________________________________________________\n☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n____________________________________________________________");
+                    }
```

Comment:
This block of code doesn't really do much. The method in the try will always throw, so it might as well have been a one-liner.